tests/slo/requirements.txt

```diff
@@ -0,0 +1,4 @@
+requests==2.28.2
+ydb==3.3.4
```

Comment:
Is use fixed version of sdk - temporary (instead of use sdk from the repo)?

Reply:
yep, needs to build container with local changes